core/services/relay/evm/event_binding.go

```diff
@@ -21,19 +21,20 @@ import (
 )
 
 type eventBinding struct {
-	address        common.Address
-	contractName   string
-	eventName      string
-	lp             logpoller.LogPoller
-	hash           common.Hash
-	codec          commontypes.RemoteCodec
-	pending        bool
-	bound          bool
-	registerCalled bool
-	lock           sync.Mutex
-	inputInfo      types.CodecEntry
-	inputModifier  codec.Modifier
-	codecTopicInfo types.CodecEntry
+	address         common.Address
+	contractName    string
+	eventName       string
+	logPollerFilter logpoller.Filter
+	lp              logpoller.LogPoller
+	hash            common.Hash
+	codec           commontypes.RemoteCodec
+	pending         bool
```

Comment:
Didn't we replace this with confidence levels at some point? Or did that PR never get merged?

Reply:
That PR is open but will be merged after this one